social_engagement/models.py

```diff
@@ -278,6 +312,7 @@ def on_studentengagementscore_save(sender, instance, created, **kwargs):
     score value in the history table, so we have a complete history
     of the student's engagement score
     """
+    instance.refresh_from_db()
```

Comment:
Yes, because we use F expressions, [instance.score](https://github.com/edx-solutions/discussion-edx-platform-extensions/pull/26/files/58aa7defa3d410a9d701e312b6c6d962a9ccd557#diff-18efce608f8be871ff23eae69c9b925dR320) will fail to be read otherwise.

Reply:
Ah! Makes sense.